Objects/moduleobject.c

```diff
@@ -624,6 +801,46 @@ PyModule_GetDict(PyObject *m)
     return _PyModule_GetDict(m);  // borrowed reference
 }
 
+int
+PyModule_GetStateSize(PyObject *m, Py_ssize_t *size_p)
+{
+    *size_p = -1;
+    if (!PyModule_Check(m)) {
+        PyErr_Format(PyExc_TypeError, "expected module, got %T", m);
+        return -1;
+    }
```

Comment:
```suggestion
    if (!PyModule_Check(m)) {
        PyErr_Format(PyExc_TypeError, "expected module, got %T", m);
        *size_p = -1;
        return -1;
    }
```

Same suggestion for PyModule_GetToken().

Reply:
This is intentional, to avoid forgetting the assignment if we add more error cases.